src/index.js

```diff
@@ -39,7 +40,11 @@ class FtpServer extends EventEmitter {
       _.get(this, 'options.pasv_min'),
       _.get(this, 'options.pasv_max'));
 
+	const timeout = Number(this.options.socketTimeout);
+	this.socketTimeout = isNaN(timeout) ? 0 : timeout;
```

Comment:
No need to set a whole new variable when we have it already set under `options`.

Reply:
Someone prefers the split version to increase readability, but it makes sense. No problem